src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -56,40 +57,43 @@ public Command parseCommand(String userInput) throws ParseException {
 
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
+            case AddCommand.COMMAND_WORD:
```

Comment:
Was the indententation changed on this?

Reply:
Yea I figured it should be indented under switch.